src/components/AssetsTable/AssetsTable.test.jsx

```diff
@@ -457,11 +463,11 @@ describe('<AssetsTable />', () => {
     it('moves from modal to trashcan on modal close', () => {
       trashButtons.at(0).simulate('click');
 
-      expect(closeButton.matchesElement(document.activeElement)).toEqual(true);
+      expect(closeButton.html()).toEqual(document.activeElement.outerHTML);
```

Comment:
@fysheets -- oh, that's how you did it! 👍

Reply:
yeah! @jaebradley helped logic this one out :) that `.outerHTML` was the key

Reply:
I'm not sure this is the best way to do it...but it works #yolo

![alt-text](https://media.giphy.com/media/y65VoOlimZaus/giphy.gif)